addons/web/static/src/js/chrome/abstract_web_client.js

```diff
@@ -128,6 +128,7 @@ var AbstractWebClient = Widget.extend(ServiceProviderMixin, KeyboardNavigationMi
                 // Listen to 'scroll' event and propagate it on main bus
                 self.action_manager.$el.on('scroll', core.bus.trigger.bind(core.bus, 'scroll'));
                 core.bus.trigger('web_client_ready');
+                odoo.isReady = true;
```

Comment:
now that I think of it, you can do this in the test itself (where you monkey patch the web client to add hooks)